src/main/java/jp/ne/paypay/model/PaymentOrderDetails.java

```diff
@@ -1,273 +0,0 @@
-package jp.ne.paypay.model;
-
-import com.google.gson.annotations.SerializedName;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
-
-import io.swagger.annotations.ApiModelProperty;
-
-/**
- * PaymentOrderDetails
- */
-
-public class PaymentOrderDetails {
```

Comment:
This is no longer used?

Reply:
No, I have removed other models also which are not used and tested the apis